package/Dockerfile

```diff
@@ -17,7 +17,10 @@ RUN apt-get update \
     && apt-get install -y \
     curl \
     jq \
-    vim
+    vim \
+    systemd \
```

Comment:
why are we installing systemd in a Docker image?

Reply:
to basically get the journalctl logs from k3s, to get related flags to different components, since there are no logs or argument file for those

Reply:
Ah, so we are installing all of systemd just to get journalctl? Is there a minimal package available for that?